bin/migrate-oats-data/submap/direction_mapping.py

```diff
@@ -10,23 +11,64 @@ def add_direction_field(data):
     data['south_land_use_type'] = None   
     return data
 
-def map_direction_field(data, dir_data):
-    if data['alr_application_id'] == dir_data['alr_application_id']:
-        if dir_data['cardinal_direction'] == 'EAST':
-            data['east_land_use_type_description'] = dir_data['description']
-            data['east_land_use_type'] = dir_data['nonfarm_use_type_code']
-        if dir_data['cardinal_direction'] == 'WEST':
-            data['west_land_use_type_description'] = dir_data['description']
-            data['west_land_use_type'] = dir_data['nonfarm_use_type_code']
-        if dir_data['cardinal_direction'] == 'NORTH':
-            data['north_land_use_type_description'] = dir_data['description']
-            data['north_land_use_type'] = dir_data['nonfarm_use_type_code']
-        if dir_data['cardinal_direction'] == 'SOUTH':
-            data['south_land_use_type_description'] = dir_data['description']
-            data['south_land_use_type'] = dir_data['nonfarm_use_type_code']
-    else:
-        return data
+def get_NESW_rows(rows, cursor):
+    # fetches adjacent land use data, specifically direction, description and type code
+    application_ids = [dict(item)["alr_application_id"] for item in rows]
+    application_ids_string = ', '.join(str(item) for item in application_ids)    
+    adj_rows_query = f"""SELECT * from 
+                        oats.oats_adjacent_land_uses oalu 
+                        WHERE oalu.alr_application_id in ({application_ids_string})
+                        """
+    cursor.execute(adj_rows_query)
+    adj_rows = cursor.fetchall()
+    return adj_rows
+
+def map_direction_values(data, direction_data):
+    # adds direction field values into data row
+    data['east_land_use_type_description'] = direction_data.get(data["alr_application_id"], {}).get('east_description', 'No entry found')
```

Comment:
Add 'No entry found' to rows that have data in other direction columns instead of leaving null. Can make None (NULL) if preffered

Reply:
changed to 'No data found'